src/components/implementation/no_interface/cFE_booter/osfiles.c

```diff
@@ -260,8 +320,8 @@ int32 OS_initfs(char *address, char *devname, char *volname, uint32 blocksize, u
 */
```

Comment:
As far as I can tell this is used to set up some space for an existing or new filesystem (on disk?).  It looks like it can also be used to initialize a preloaded filesystem.  We don't need to do anything special to prepare for an existing fs to be mounted, or a new fs to be created.  So if it is called there is no reason to return anything other than success in case an application relies on it returning successful.